build.gradle

```diff
@@ -1,6 +1,7 @@
 buildscript {
     repositories {
         jcenter()
+        mavenCentral()
```

Comment:
This is redundant since jcenter is a superset of maven central, I'll take it out